shap/explainers/_tree.py

```diff
@@ -1476,7 +1478,12 @@ def buildTree(index, node):
                     self.features[vleaf_idx] = -1
                     self.thresholds[vleaf_idx] = -1
                     self.values[vleaf_idx] = [vertex["leaf_value"]]
-                    self.node_sample_weight[vleaf_idx] = vertex["leaf_count"]
+                    # FIXME: "leaf_count" currently doesn't exist if we have a stump tree.
+                    # We should be technically be assigning the number of samples used to
+                    # train the model as the weight here, but unfortunately this info is
+                    # currently unavailable in `tree`, so we set to 0 first.
```

Comment:
Will using `0` as a placeholder be problematic if this variable is used as a weight?

Reply:
I believe the functions in `tree_shap.h` have already taken the 0's into account.

Also, (1) the `node-sample_weights` are instantiated as an array of 0's in general, and (2) tests are passing, so we should be good. :)